eclair-core/src/main/scala/fr/acinq/eclair/channel/MetaCommitments.scala

```diff
@@ -38,19 +38,21 @@ case class Common(localChanges: LocalChanges, remoteChanges: RemoteChanges,
                   originChannels: Map[Long, Origin], // for outgoing htlcs relayed through us, details about the corresponding incoming htlcs
                   remoteNextCommitInfo: Either[WaitForRev, PublicKey], // this one is tricky, it must be kept in sync with Commitment.nextRemoteCommit_opt
                   remotePerCommitmentSecrets: ShaChain) {
+  val nextRemoteCommitIndex = remoteCommitIndex + 1
+
   /**
    * When reconnecting, we drop all unsigned changes.
    */
-  def discardUnsignedUpdates(implicit log: LoggingAdapter): Common = {
+  def discardUnsignedUpdates()(implicit log: LoggingAdapter): Common = {
```

Comment:
I don't think we should add parentheses, as there is no side effect?

Reply:
It's very weird without parenthesis, `discardXXX` really feels like it's _doing_ something, not just _accessing_ something...

Reply:
I put them back in 612b327a8.